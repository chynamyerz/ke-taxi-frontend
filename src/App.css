.App {
  text-align: center;
}
.topHeader {
  padding-left: 10px;
  padding-right: 10px;
}
.toolbar-title {
  margin-left: auto;
  margin-right: auto;
}
.ke-Taxi {
  margin-top: 2%;
  text-align: center;
  display: block;
}
