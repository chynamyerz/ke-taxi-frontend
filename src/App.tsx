import {
  IonApp,
  IonButtons,
  IonContent,
  IonHeader,
  IonMenuButton,
  IonPage,
  IonTitle,
  IonToolbar
} from "@ionic/react";
import React from "react";
import { Link } from "react-router-dom";
import "./App.css";
import Logo from "./assets/images/Group3.svg";
import signin from "./assets/images/signin.svg";
import Navigation from "./navigation/Navigation";
import Routes from "./Routes";

const App: React.FC = () => {
  return (
    <IonApp>
      <IonHeader slot={"start"}>
        <IonToolbar>
          <IonButtons slot={"start"} style={{ paddingLeft: "10px" }}>
            <IonMenuButton></IonMenuButton>
          </IonButtons>
          <Link to={"/"} style={{ textDecoration: "none", color: "black" }}>
            <IonTitle style={{ marginLeft: "auto", marginRight: "auto" }}>
              <img src={Logo} alt="Ke-Taxi" />
            </IonTitle>
          </Link>
          <Link
            to={"/signin"}
            style={{ textDecoration: "none", color: "black" }}
            slot={"end"}
          >
            <IonButtons style={{ paddingRight: "10px" }}>
              <img src={signin} alt="signin" />
            </IonButtons>
          </Link>
        </IonToolbar>
      </IonHeader>

      <IonPage style={{ marginTop: "50px" }}>
        <Navigation />
        <IonContent scrollY={false} id={"main"}>
          <Routes />
        </IonContent>
      </IonPage>
    </IonApp>
  );
};

export default App;
