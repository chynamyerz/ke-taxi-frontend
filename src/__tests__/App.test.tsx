// tslint:disable-next-line:no-implicit-dependencies
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import React from "react";
import App from "../App";

describe("App", () => {
  it("renders without crashing", () => {
    const wrapper = shallow(<App />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
