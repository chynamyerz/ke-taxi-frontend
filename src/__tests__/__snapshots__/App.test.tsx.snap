// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders without crashing 1`] = `
<ForwardRef(IonApp)>
  <ForwardRef(IonHeader)
    slot="start"
  >
    <ForwardRef(IonToolbar)>
      <ForwardRef(IonButtons)
        slot="start"
        style={
          Object {
            "paddingLeft": "10px",
          }
        }
      >
        <ForwardRef(IonMenuButton) />
      </ForwardRef(IonButtons)>
      <Link
        style={
          Object {
            "color": "black",
            "textDecoration": "none",
          }
        }
        to="/"
      >
        <ForwardRef(IonTitle)
          style={
            Object {
              "marginLeft": "auto",
              "marginRight": "auto",
            }
          }
        >
          <img
            alt="Ke-Taxi"
            src="Group3.svg"
          />
        </ForwardRef(IonTitle)>
      </Link>
      <Link
        slot="end"
        style={
          Object {
            "color": "black",
            "textDecoration": "none",
          }
        }
        to="/signin"
      >
        <ForwardRef(IonButtons)
          style={
            Object {
              "paddingRight": "10px",
            }
          }
        >
          <img
            alt="signin"
            src="signin.svg"
          />
        </ForwardRef(IonButtons)>
      </Link>
    </ForwardRef(IonToolbar)>
  </ForwardRef(IonHeader)>
  <IonPage
    style={
      Object {
        "marginTop": "50px",
      }
    }
  >
    <Navigation />
    <ForwardRef(IonContent)
      id="main"
      scrollY={false}
    >
      <Routes />
    </ForwardRef(IonContent)>
  </IonPage>
</ForwardRef(IonApp)>
`;
